contracts/L1GlobalForkRequester.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.20;
+
+/* 
+   Contract to proxy a fork request from the bridge to its ForkingManager
+   Any L2 contract can call us over the bridge to get us to request a fork.
+   We record the dispute they were forking over
+*/
+
+// TODO: An alternative would be to look up the ForkingManager on L2ChainInfo and call it directly.
+// TODO: We could use this to manage the dispute data, in which case we could gate initiateFork() to only be callable by us.
+// TODO: The whitepaper implies a whitelist about who is allowed to fork us. Currently anybody can as long as they pay.
```

Comment:
I would keep it separated. Separation of concerns is always good and I don't see an attack vector by allowing anyone to call it.